src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6736,6 +6736,21 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_min_20_boolean) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_string_input) {
+    const auto model = convert_model("string_input.onnx");
+    auto test_case = test::TestCase(model);
+    test_case.add_input<std::string>({"string1", "string2"});
+    test_case.add_expected_output<int64_t>({2});
+
```

Comment:
also recommend to add a test with Parameter (string format)->Result to check string output, also.

Reply:
may be it makes check Constant -> Result as well, if possible